api-tests/api.test.js

```diff
@@ -138,6 +146,54 @@ test('/events endpoint', async () => {
   })
 })
 
+test('/events create endpoint', async () => {
+  expect(config.seedUser.roles.isAdmin).toBe(true)
+
+  const event = {
+    key: '1970flir',
+    name: 'FLIR x Daimler',
+    district: 'pnw',
+    week: 4,
+    startDate: '1970-01-01T19:46:40-08:00',
+    endDate: '1970-01-02T09:40:00-08:00',
+    location: {
+      name: 'Cleveland High School',
+      lat: 45.498555,
+      lon: -122.6385231,
+    },
+    webcasts: [
+      {
+        type: 'twitch',
+        url: 'https://www.twitch.tv/firstwa_red',
+      },
+    ],
+  }
+
+  const response = await fetch(addr + '/events', {
+    method: 'PUT',
+    body: JSON.stringify(event),
+    headers: {
+      'Content-Type': 'application/json',
+      Authentication: 'Bearer ' + (await getJWT()),
+    },
+  })
+
+  expect(response.status).toBe(201)
+
+  const d = await fetch(addr + `/events/${event.key}/info`).then(d => d.json())
+
+  expect(d.data).toEqual({
+    key: event.key,
+    name: event.name,
+    district: event.district,
+    week: event.week,
+    startDate: expect.toBeEqualDates(event.startDate),
```

Comment:
Nit again: I feel like this name doesn't make much sense for a normal matcher _or_ an asymmetric matcher.

```ts
expect(startDate).toBeEqualDates("2018-blah")
```
```ts
expect(info).toEqual({
  date: expect.toBeEqualDates("2018-blah")
}
```

Would `toEqualDate` be more readable?

Reply:
Yeah. I just was confused about the verbiage because when you write the function it takes two args but it's really more like a method on an object. That makes more sense.